testgrid/config.yaml

```diff
@@ -6654,6 +6656,11 @@ dashboards:
     test_group_name: post-test-infra-push-prow
     code_search_url_template:
       url: https://github.com/kubernetes/test-infra/compare/<start-custom-0>...<end-custom-0>
+  - name: deploy-prow
+    description: deploys the configured version of prow by running deploy.sh
+    test_group_name: post-test-infra-deploy-prow
+    code_search_url_template:
+      url: https://github.com/kubernetes/test-infra/compare/<start-custom-0>...<end-custom-0>
```

Comment:
Should we add an alert?

Reply:
I don't think so, too slow.

Ideally we provide a way to report back to the PR with the job it kicked off